### PR TITLE
Add bubble sort method

### PR DESCRIPTION
**Pull request by yigitm**
Build the repository skeleton.

 Build the method structure.

 Verify the output

 Fix linter errors

### CODE REVIEW

**Comment by adaorachi** [comment]
### Hi @yigitm and @AlduLonghi :wave: 

Well done on the completion of the first draft of this project. Your README content looks pretty detailed. :muscle: :100: 
Your bubble_sort methods are well implemented but there is a couple of change you need to do for the methods to work well and pass all test suite cases.
So hang in there, you are just a step away from completing this project.

---

### CHANGES REQUIRED :recycle:

---

File: bubble_sort.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      if arr[i] > arr [i + 1]
+        (arr[i], arr[i + 1] = arr[i + 1], arr[i])
+      else arr = arr
+      end
+      i += 1
+    end
+  end
+  prints arr
+end
+
```
Comment (adaorachi) [inline bubble_sort.rb]: - [ ] In methods or functions that involve all logic, you do not print/puts the result, instead, you return the result.
In the highlighted code above, you need to return the output and remember that Ruby implicitly returns the last variable in a method. So you do not need to add the return statement.
**This should be done for both methods.**

Note: If you wish to see the result of your output in the console, you can add the puts statement when you call the method. Like so -
`puts bubble_sort([2, 7, 5, 1, 10, 5, 39, 20, 6, 8, 17])`
```
sorted = bubble_sort_by([2, 7, 5, 6, 1]) do |left, right|
  left - right
end
puts sorted
```

**Review by adaorachi — CHANGES_REQUESTED** [review]

**Comment by adaorachi** [comment]
- [ ] There are two contributors to this project and there should be equal commits from both but I only see commits from just one contributor. Kindly ensure you improve that in subsequent commits.

---

### After implementing the changes above, you can request another code review

---

Should you have any question, please do reach out to me on Slack with the username **@adaorachi**

Happy Coding :relaxed: 

**MaryAnn Chukwuka**

**Review by rOluochKe — APPROVED** [review]
Hello Team
### Well done :clap: You got it right this time. The project has been approved. Congratulations :boom: :100:
### Merge the project and submit it. Good luck with the next one!

- Happy coding!

**Pull request merged**